tiingo/api.py

```diff
@@ -88,7 +90,11 @@ def get_ticker_metadata(self, ticker):
         """
         url = "tiingo/daily/{}".format(ticker)
         response = self._request('GET', url)
-        return response.json()
+        if fmt=='json':
+            return response.json()
+        elif fmt=='object':
+            # inspired by https://stackoverflow.com/a/15882054
+            return json.loads(json.dumps(response.json()), object_hook=lambda d: namedtuple('X', d.keys())(*d.values()))
```

Comment:
can we store `data = response.json()` once, and then use it across either branch of the `if` statement? Also, aim to use a name like 'Ticker' instead of `X`.